core/src/main/java/org/elasticsearch/action/admin/indices/template/put/PutIndexTemplateRequest.java

```diff
@@ -286,7 +296,20 @@ public PutIndexTemplateRequest source(Map templateSource) {
         for (Map.Entry<String, Object> entry : source.entrySet()) {
             String name = entry.getKey();
             if (name.equals("template")) {
-                template(entry.getValue().toString());
+                // This is needed to allow for bwc (beats, logstash) with pre-5.0 templates (#21009)
```

Comment:
@nik9000 I assume you mean here pre-6.0 templates. Or will there be support for `indext-patterns` in the 5.0 releases?